flixel/text/FlxText.hx

```diff
@@ -840,6 +840,8 @@ class FlxText extends FlxSprite
 			applyBorderTransparency();
 			applyFormats(_formatAdjusted, false);
 			
+			avoidBlurry();
```

Comment:
@gamedevsam I didn't checked because in some cases `wordWrap` is enabled but is not affecting the text. And in this cases this method can avoid the blurry lines.